README_CN.md

```diff
@@ -177,9 +177,10 @@ Dashboard 默认允许任何 IP 访问。你可以自行修改 `conf/config.yaml
 
 ## 视频和幻灯片
 
-- [APISIX 的选型、测试和持续集成](https://www.upyun.com/opentalk/432.html)
-
-- [APISIX 高性能实践](https://www.upyun.com/opentalk/429.html)
+- 2019.10.30 [Apache APISIX 微服务架构极致性能架构解析](https://www.upyun.com/opentalk/440.html) .
+- 2019.8.31 [APISIX 技术选型、测试和持续集成](https://www.upyun.com/opentalk/433.html) .
+- 2019.8.31 [APISIX 高性能实战2](https://www.upyun.com/opentalk/437.html) .
+- 2019.7.6 [APISIX 高性能实战(Chinese)](https://www.upyun.com/opentalk/429.html) .
```

Comment:
do you have the text version?

Reply:
I think I can find some of them. If you have a list, please paste it.

Reply:
These links already contain text content ^_^